doc/news/cvmetadata2.rst

```diff
@@ -0,0 +1,7 @@
+**Added:**
+
+* Added properties to extract `tags` describing the data, `figure_label` and `curve_label` from the SVG, which are included in the property `metadata`.
```

Comment:
```suggestion
* Added property `CV.tags`, `CV.figure_label`, and `CV.curve_label` which are extracted from `<text>` elements in the SVG. The extracted values are stored in `CV.metadata`.
```

I didn't understand what you did until I read the code. Maybe it's easier to understand now?

Reply:
Looks good to me.